Strange javac "automatic module" warning for `benchmark-jmh`

### Description

I suddenly noticed this warning when running top-level `./gradlew jar` in Lucene `main`:

```
/s1/l/trunk/lucene/benchmark-jmh/src/java/module-info.java:20: warning: requires directive for an automatic module
  requires jmh.core;
              ^
1 warning
```

I think it is related to java modules but I'm not sure how to fix/suppress it :)

--- a/lucene/benchmark-jmh/src/java/module-info.java
+++ b/lucene/benchmark-jmh/src/java/module-info.java
@@ -16,6 +16,9 @@
  */
 
 /** Lucene JMH benchmarks. */
+
+// jmh.core is not modularized and causes a warning. Suppressing it until it is modularized.
+@SuppressWarnings("requires-automatic")
 module org.apache.lucene.benchmark.jmh {
   requires jmh.core;
   requires jdk.unsupported;

